lang/python27/Portfile

```diff
@@ -71,6 +71,12 @@ if {${subport} eq ${name}} {
     openssl.branch  1.1
 }
 
+configure.args-append \
+                    --enable-framework=${frameworks_dir} \
```

Comment:
`frameworks_dir` won't have the correct value for python27-bootstrap if used here.

Reply:
Yeah - that is why I did it the way I originally did it, to avoid redoing the whole logic... I'll see if there is some way to straighten this out as you want it...